playbook/buildfarmer/ultra_scripted_buildfarm_day.md

```diff
@@ -75,15 +69,36 @@ For each test regression check the log output in Jenkins, find the root cause, a
 For gathering information about the test regressions:
 
 ```bash
-cd ~/buildfarm/database/scripts
+cd ~/buildfarm-tools/database/scripts
 ./sql_run.sh errors_get_first_time.sql [error_name] # Check the first 25 appearances of the test regression
 ./sql_run.sh errors_get_last_ones.sql [error_name] # Check the last 25 appearances of the test regression
 ./sql_run.sh calculate_flakiness_jobs.sql [error_name] [time_range] # Check the ratio of the test regression per job in the given time range
 ```
 
 See [Check test regressions](./scripted_buildfarm_day.md#5-check-test-regressions-on-buildfarm-dashboards) for more information.
 
-### 6. Check for remaining warnings
+
+## 6. Add known issues to the buildfarmer database
+
+For each issue you opened:
+```bash
+./issue_save_new.sh "<test-name>" "<package-name>" "<job-name>" "<github-issue-url>"
+```
+
+> Details in [Scripted buildfarm day: Add known issues](./scripted_buildfarm_day.md#6-add-known-issues-to-the-buildfarmer-database) and [Known issues: Add known issues](./known_issues.md#add-a-known-issue-test-failures)
+
+If any change was made to the buildfarmer database, push the changes:
+
+> **Note**:
+> This is part of the buildfarmer role. If you are not an official buildfarmer, you should suggest your changes in [buildfarm-tools discussions](https://github.com/osrf/buildfarm-tools/discussions)
+
```

Comment:
@Crola1702 Please add to your backlog of tasks testing what happens if an external user tries to push to the repo database.